packages/react/src/Link/Link.module.css

```diff
@@ -1,13 +1,28 @@
 .Link {
-  --arrow-width: var(--base-size-20);
+  --visual-width: var(--external-icon-size, var(--arrow-size, 0));
```

Comment:
is the default supposed to be 0 here or var(--base-size-20)?

Reply:
The default value is `0`. This allows us to set the underline width in one single place. So, when there is no visual (no arrow or external link icon), nothing is deducted from the total width (the result is `100%`).